airflow/utils/dag_processing.py

```diff
@@ -984,7 +984,7 @@ def _log_file_processing_stats(self, known_file_paths):
             file_name = os.path.basename(file_path)
             file_name = os.path.splitext(file_name)[0].replace(os.sep, '.')
             if last_runtime:
-                Stats.gauge(
+                Stats.timing(
```

Comment:
actually, the stat calculates the time for the dag processing time in the latest/last iteration. If the data changes, the stat will change. So I think gauge works in this case. Could you share why you think gauge doesn't work?

Reply:
ah, i guess we are interested in tracking how long it takes to process a dag over time. so we can see if dag processing is slowing down over time

Reply:
maybe i don't fully understand how the scheduler works and what those metrics were counting. i think i understand why it's a gauge here. Maybe i need to add the metric we're interested in in another place. any thoughts? we want to see a line chart that shows if dag processing is taking a while to help alert if the scheduler is overwhelmed.

Reply:
+1, I think both stats are valid.

Reply:
@andrewhharmon actually I have a open pr which will instrument the dag loading/processing time. Is that what you want?

Reply:
https://github.com/apache/airflow/pull/5350

Reply:
yeah, that looks perfect. we can delete my PR. Thanks!